.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,47 +1,35 @@
-<!-- Please read these guidelines before opening your PR: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
-<!-- The text between the arrows are comments - they will not be visible on your PR. -->
+<!-- Guidelines: https://docs.spacestation14.io/en/getting-started/pr-guideline -->
 
 ## About the PR
-<!-- What did you change in this PR? -->
+<!-- What did you change? -->
 
 ## Why / Balance
-<!-- Why was it changed? Link any discussions or issues here. Please discuss how this would affect game balance. -->
+<!-- Why was it changed? Link any relevant discussions or issues. -->
 
 ## Technical details
-<!-- If this is a code change, summarize at high level how your new code works. This makes it easier to review. -->
+<!-- Summary of code changes for easier review. -->
 
 ## Media
-<!-- 
-PRs which make ingame changes (adding clothing, items, new features, etc) are required to have media attached that showcase the changes.
-Small fixes/refactors are exempt.
-Any media may be used in SS14 progress reports, with clear credit given.
-
-If you're unsure whether your PR will require media, ask a maintainer.
--->
+<!-- Attach media if the PR makes ingame changes (clothing, items, features, etc). 
+Small fixes/refactors are exempt. Media may be used in SS14 progress reports with credit. -->
 
 ## Requirements
-<!-- 
-Due to influx of PR's we require to ensure that PR's are following the correct guidelines.
-
-Please take a moment to read these if its your first time.
-
-Check the boxes below to confirm that you have in fact seen these (put an X in the brackets, like [X]):
--->
-- [ ] I have read and I am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html). I understand that not doing so may get my pr closed at maintainer’s discretion
-- [ ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase
+<!-- Confirm the following by placing an X in the brackets [X]: -->
+- [ ] I have read and am following the [Pull Request Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).
+- [ ] I have read and I am following the [Changelog Guidelines](https://docs.spacestation14.io/en/getting-started/pr-guideline#changelog)
+- [ ] I have added media to this PR or it does not require an ingame showcase.
+- [ ] I understand that not following the above may get my pr closed at maintainer’s discretion
```

Comment:
this last bullet should just be a comment. on this section. doesn't really need to be visible.